docs/DeveloperGuide.md

```diff
@@ -310,20 +310,70 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
-### \[Proposed\] Reading an individual event feature
+### Reading an individual event feature
 
-#### Proposed Implementation
+#### Implementation
+
+When event show command is called, a pop-up window (an `EventShowWindow` object) will show, displaying the information of the event that corresponds to the given index. The event to show is obtained from the `eventToShowList` instantiated in `ModelManager`.
+
+Given below is an example usage scenario and how the event show mechanism behaves at each step:
+
+Step 1. The user launches the app for the first time, and executes `ecreate n/first event ...` to create a new event. This called method `Model#updateFilteredEventList()` to update both the `filteredEventList` and the `eventToShowList`.
```

Comment:
`This called method` --> `This calls the method`

Reply:
yes english dashen